src/run_tribler.py

```diff
@@ -25,7 +29,7 @@ def show_error(exc: Exception, shutdown: bool = True) -> None:
             import subprocess
 
             subprocess.Popen(['xmessage', '-center', text])  # noqa: S603, S607
-        elif sys.platform == 'Darwin':
+        elif sys.platform == 'darwin':
```

Comment:
Maybe also change `Linux` to `linux` while you're at it? I guess the reason for the confusion is that the result of `platform.system()` starts with a capital letter, and `sys.platform` does not.

Reply:
Sure, done 👌